components/brave_welcome_ui/containers/screens/shieldsBox.tsx

```diff
@@ -30,9 +32,17 @@ export default class ShieldsBox extends React.PureComponent<Props> {
       >
         <WelcomeShieldsImage />
         <Title>{getLocale('privacyTitle')}</Title>
-        <Paragraph
-          dangerouslySetInnerHTML={{ __html: getLocale('privacyDesc') }}
-        />
+        <Paragraph>
+          {text[0]}
+          <a href='https://brave.com/p3a'
+             target='_blank'
+             rel='noopener noreferrer'>{text[1]}</a>
+          {text[2]}
+          <a href='chrome://settings/privacy'
+             target='_blank'
+             rel='noopener noreferrer'>{text[3]}</a>
+          {text[4]}
+        </Paragraph>
```

Comment:
to test this: go to brave://welcome and make sure that the links in the 3rd page (about p3a) work correctly